CodeGen/kubernetes/manifests/gaudi/codegen.yaml

```diff
@@ -17,9 +17,9 @@ data:
   TGI_LLM_ENDPOINT: "http://codegen-tgi"
   HUGGINGFACEHUB_API_TOKEN: "insert-your-huggingface-token-here"
   HF_HOME: "/tmp/.cache/huggingface"
-  http_proxy:
-  https_proxy:
-  no_proxy:
+  http_proxy: ""
+  https_proxy: ""
+  no_proxy: ""
```

Comment:
This issue is being addressed in PR  https://github.com/opea-project/GenAIExamples/pull/630